### PR TITLE
Hello World

### PR DESCRIPTION
**Pull request by Sboursen**
Initial commit

### CODE REVIEW

**Comment by Sboursen** [comment]
Hi @Sboursen 😃, 

Good start 👏 
You've done well adding the `HTML` and the `CSS` files, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (Sboursen) [inline index.html:5]: - [x] The page needs a good descriptive title.
- Suggestion: you can put `Hello world` as a title.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <link rel="stylesheet" href="css/style.css" />
+    </head>
+
+    <body>
+        <div>“Hello World, we are software developers!”</div>
```
Comment (Sboursen) [inline index.html:10]: - [x] Try to use semantic `HTML` tags whenever is possible.
- suggestion: For this case, you can use the `<h1> </h1>` tag, as this is the first header of your page.

**Comment by Sboursen** [comment]
### There are additional changes We would like you to do:

- [ ] Please set up the [HTML & CSS Linters](https://github.com/microverseinc/linters-config/tree/master/html-css) in your repository. After that, make sure you don't have any linter errors.
- [ ]  Please add a `.gitignore` file to your repository. This allows you to ignore all the files that you don't wish to commit and push to your remote repo.
- [ ]  Please document your repository by using this [README.md](https://github.com/microverseinc/readme-template/blob/master/README.md) template.
- [ ] Please add a style to your `<h1> </h1>` in your `CSS` file.
- Suggestion: just add `color: ;` of your preference to the CSS file.
- [OPTIONAL]: You did great following the `Github flow`. One improvement you can do is to use action verbs at the start of your commits.

**Review by ReshyResh — APPROVED** [review]
# Hi @Sboursen , 

### What a great review! 
You did an amazing job:
  - ✔Requested changes have good explanations as to why they're requested.
  - ✔The requests are polite suggestions and not orders.
  - ✔Clear and easy to understand comments.
 
Your project is complete, you can now merge it! Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**